assets/img/standard-base-documentation/README.md

```diff
@@ -4,8 +4,10 @@ The current illustration is a digital remake of this [original visual](/patterns
 If you want to edit this illustration, please request access to this [source document](https://docs.google.com/presentation/d/11JOByEO9QXlRLXX5BIv9rjUzPzCKErZzknD1OLcprQQ/edit?usp=sharing).
```

Comment:
Do you have an email address that I can use to add you as Editor to this [Google Doc](https://docs.google.com/presentation/d/11JOByEO9QXlRLXX5BIv9rjUzPzCKErZzknD1OLcprQQ/edit#slide=id.g15a686b4ea4_0_70)?

Then we could add your illustration there, to make it easier to edit it in the future.

Also I might try to change formatting a tiny bit, to make it even more consistent with the other two visuals that we already have. (What I have in mind are  coloring and centering the text below the image).